.travis.yml

```diff
@@ -9,19 +9,23 @@ os:
   - osx
 
 install:
+  # Install Miniconda
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
```

Comment:
```suggestion
      wget https://repo.continuum.io/miniconda/Miniconda3-4.5.4-MacOSX-x86_64.sh -O miniconda.sh;
```
This Miniconda version is based on Python 3.6. This is to avoid a potential problem related to missing DLLs that can appear during the build and its subsequent test.